contracts/Overtime/SportsAMMV2.sol

```diff
@@ -714,6 +856,18 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
         emit SetLiquidityPool(_liquidityPool);
     }
 
+    /// @notice sets new LP Pool with LP address and the supported collateral
+    /// @param _collateralAddress collateral address that is supported by the pool
+    /// @param _liquidityPool new LP address
+    function setCollateralLiquidityPool(address _collateralAddress, address _liquidityPool) external onlyOwner {
```

Comment:
please name it setLiquidityPoolForCollateral

Reply:
done